kotlin/query-with-cql-filters/README.md

```diff
@@ -0,0 +1,40 @@
+# Query with CQL filters
+
+Query data from an OGC API feature service using CQL filters.
+
+![Image of Query with CQL Filters](query-with-cql-features.png)
```

Comment:
could the preview image show the UI?

Reply:
It would be challenging to show the UI on landscape mode, since it will cover the `MapView`. Most screenshot's on Android show the map which is why I used it as the preview image.